restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -502,6 +538,12 @@ public Response patchDocPath(
           @NotNull(message = "payload not provided")
           @NotBlank(message = "payload must not be empty")
           String payload,
+      @ApiParam(
+              value =
+                  "Include this to put a time-to-live on the document in the data store; use 'auto' to make the TTL match that of the parent document.",
+              required = false)
+          @QueryParam("ttl")
+          String ttl,
```

Comment:
Well this is confusing, it's `int` or `string`, I am not sure if this is really the best way.. how would this be complicated from the user perspective? Actually in HTTP query params are always strings right?

Reply:
yea to the user all params are strings